api/src/main/resources/marquez/db/migration/V66__job_versions_io_mapping_add_job_reference.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE job_versions_io_mapping ADD COLUMN job_uuid uuid REFERENCES jobs(uuid) ON DELETE CASCADE;
+
+CREATE INDEX job_versions_io_mapping_job_uuid ON job_versions_io_mapping (job_uuid);
+
+UPDATE job_versions_io_mapping
```

Comment:
Do you think it'd be helpful to split the `ALTER` and `UPDATE` ... `SET` into separate migrations? That way, the migration for `ALTER` and new index will be separate from the migration to set the new `job_uuid ` col (which may or may not take a long time to compelete).

Reply:
yeah, added that. I think this deserves an extra test (added todo within the pr for that)